samples/04_gis_analysts_data_scientists/export_video_using_arcgis_api_for_python.ipynb

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
<p>Rephrase "This notebooks intends to showcase the capabilities of ArcGIS Python API to create a movie based on the area of extent provided." to "This notebook creates a movie to visualize the shriking of .... lake"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>done</p>